pkg/cgroups/fsios.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This module provides the native implementation of the filesystem
+// interface (fsi).
+
+package cgroups
+
+import (
+	"os"
+	"path/filepath"
+)
+
+type fsOs struct{}
+
+type osFile struct {
+	file *os.File
+}
+
+func NewFsiOS() fsiIface {
```

Comment:
Unexport `NewFsiOS` or export `fsiIface`(?)

Reply:
Thanks! Stumbling with the visibility still... :)

Decided to keep the filesystem interface private to the cgroups. It'll be easier to make it public later on than the other way around. :D